pkg/meta/base.go

```diff
@@ -1977,3 +1994,58 @@ func (m *baseMeta) ScanDeletedObject(ctx Context, tss trashSliceScan, pss pendin
 	}
 	return eg.Wait()
 }
+
+func (m *baseMeta) Clone(ctx Context, srcIno, dstParentIno Ino, dstName string, cmode uint8, cumask uint16, count, total *uint64) syscall.Errno {
```

Comment:
Can we move these refactor to another PR?

Reply:
refactor depends on this PR, which will come out after this merger.